hydrocarbon_problem/env/env_test.py

```diff
@@ -0,0 +1,47 @@
+import numpy as np
+
+from hydrocarbon_problem.env.env import AspenDistillation
+from hydrocarbon_problem.api.fake_api import FakeDistillationAPI
+
+def make_fake_agent(env: AspenDistillation):
+    def fake_agent(obs):
+        del(obs)
+        discrete_spec, continuous_spec = env.action_spec()
+        discrete_action = np.random.randint(0, discrete_spec.num_values, size=())
+        continuous_action = np.random.uniform(low=continuous_spec.minimum,
+                                              high=continuous_spec.maximum,
+                                              size=continuous_spec.shape)
+        return discrete_action, continuous_action
+    return fake_agent
+
+def test():
+    api = FakeDistillationAPI()
```

Comment:
This is where you could plug in the AspenAPI to check that it works with the `env` file